.gitignore

```diff
@@ -1,3 +1,7 @@
 .vscode/
 __pycache__
-*.pyc
\ No newline at end of file
+*.pyc
+

```

Comment:
@BeepedWhenIShouldaBooped don't add tmp, bin, bytes files into gitignore. gitignore is mostly for things which are produced by your development work, like caches, editor config etc.

Reply:
I removed the changed (well, corrected the file and recommitted it)